__tests__/molecule.js

```diff
@@ -66,6 +66,22 @@ describe('Molecule', () => {
     expect(svg).toContain('stroke-width="2"');
   });
 
+  it('toSVG wrong coordinates (all 0)', () => {
```

Comment:
This test fails with the current version?

Reply:
Yes